mix/FileIo.cpp

```diff
@@ -101,3 +111,76 @@ bool FileIo::fileExists(QString const& _url)
 	QFile file(url.path());
 	return file.exists();
 }
+
+QStringList FileIo::compress(QString const& _deploymentFolder)
+{
+
+	Json::Value manifest;
+	Json::Value entries(Json::arrayValue);
+
+	QUrl folder(_deploymentFolder);
+	QString path(folder.path());
+	QDir deployDir = QDir(path);
+
+	dev::RLPStream str;
+	int k = 1;
+	for (auto item: deployDir.entryInfoList(QDir::Files))
+	{
+		QFile qFile(item.filePath());
+		if (qFile.open(QIODevice::ReadOnly | QIODevice::Text))
+				k++;
+	}
+	str.appendList(k);
+
+	for (auto item: deployDir.entryInfoList(QDir::Files))
+	{
+		QFile qFile(item.filePath());
+		if (qFile.open(QIODevice::ReadOnly | QIODevice::Text))
+		{
+			QFileInfo i = QFileInfo(qFile.fileName());
+
+			QByteArray _a = qFile.readAll();
+			bytes b = bytes(_a.begin(), _a.end());
+			str.append(b);
+
+			Json::Value f;
+			f["path"] = "/"; //TODO: Manage relative sub folder
```

Comment:
"/" should be the path for index.html. All other files should default to filename